### PR TITLE
Feat: calculation price

### PR DESCRIPTION
**Pull request by arlindlinum**

### CODE REVIEW

File: components/windows/swap-widget.tsx [hunk 1]
```diff
@@ -29,6 +31,32 @@ export const SwapWidget = () => {
...
+        const usdValue = ethAmountValue * ethPrice;
+        const tokens = usdValue / psyDAOTokenPrice;
+        if (!isNaN(tokens)) {
+          setTokenAmount(tokens.toFixed(2));
+        } else {
+          setTokenAmount("0.00");
+        }
+      }
+    }
+  }, [ethAmount, ethPrice, isSwapped, tokenAmount]);
```
Comment (saksijas) [inline components/windows/swap-widget.tsx:58]: Can we move this into a function that will return a value.
Both of codes look similar L39-L46 and L48-L55.
Lets refactor it a bit :)

File: constants/psyTokenPrice.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const psyDAOTokenPrice = 0.1;
```
Comment (saksijas) [inline constants/psyTokenPrice.ts:1]: can price be updated? If yes, why not put it to ENV and export env into const?

Comment (arlindlinum) [reply constants/psyTokenPrice.ts:1]: From meetings that we had they told us token sale price will stay 0.1 so that's why I used a const

Comment (saksijas) [reply constants/psyTokenPrice.ts:1]: alrighty

**Review by saksijas — APPROVED** [review]

**Pull request merged**